pytorch_lightning/accelerators/accelerator.py

```diff
@@ -75,7 +75,6 @@ def auto_device_count() -> int:
     def is_available() -> bool:
         """Detect if the hardware is available."""
 
-    @staticmethod
-    @abstractmethod
-    def name() -> str:
```

Comment:
I don't get the reasoning for removing the `name`.

If we kept it, the registry could use it to automatically define the name for the class

Reply:
Related discussion https://github.com/PyTorchLightning/pytorch-lightning/pull/12180#discussion_r818023970